pilot/proxy/envoy/testdata/cds-circuit-breaker.json.golden

```diff
@@ -100,26 +99,13 @@
     }
    },
    {
-    "name": "out.12a577b4dd04c9a3fed313fc71fb850e7f9c345f",
-    "service_name": "world.default.svc.cluster.local|redis",
-    "connect_timeout_ms": 1000,
-    "type": "sds",
```

Comment:
? why nuke?

Reply:
it isn't nuked. my change altered the ordering and it got moved down here: https://github.com/istio/istio/pull/1344/files#diff-99b71f016ec8a2c4cc3db2e721464b3dR265